src/store/channels/saga.ts

```diff
@@ -142,6 +146,7 @@ export function* roomUnfavorited(action) {
 }
 
 export function* saga() {
+  yield call(openFirstConversation);
```

Comment:
@developerfred this won't make this call upon first load. In order to do that, you should probably look into the page-load saga.

Reply:
ok, cool!